src/lua/LuaShipDef.cpp

```diff
@@ -21,6 +21,29 @@
  * The name of the ship type
  */
 
+/*
+ * Attribute: i18n
+ *
+ * The capitalized ship name that pairs with a suffix to address a ship in
+ * it's correct singular form in the translated names in /data/lang/ships.
+ *
+ * SHIP			- ship, base form.
+ * SHIP_DEF		- the ship, definitive form.
+ * SHIP_INDEF	- a ship, indefinite form.
+ *
+ * Example:
+ *
+ * > local ship = ShipDef[ad.shipid].i18n_key            -- 'NATRIX'
+ * >
+ * > local ship_def = ls[ship .. "_DEF"],                -- 'NATRIX_DEF' - 'the Natrix'
+ * > print("We're counting on " .. ship_def .. " to give us some resistance!)
```

Comment:
Nitpick-iest of nitpicks: missing closing quotation mark here. I only mention it because someone new to programming may try to copy-paste from the docs while writing a module.